tests/core/pyspec/eth2spec/test/helpers/random.py

```diff
@@ -15,6 +15,8 @@ def set_some_new_deposits(spec, state, rng):
             continue
         if rng.randrange(num_validators) < num_validators // 10:
             mock_deposit(spec, state, index)
+            if is_post_altair(spec):
+                state.inactivity_scores[index] = 0
```

Comment:
i have to wonder how scalable this approach is to check which features a fork supports and then adjusting the functionality of a helper function (e.g. `mock_deposit`) but this same pattern is used for the attestation participation randomization here so i followed it for now...

Reply:
agree